apps/web/vibes/soul/components/dropdown/index.tsx

```diff
@@ -61,7 +61,9 @@ export const Dropdown = function Dropdown({
                   'text-foreground': selectedItem === item,
                 }
               )}
-              onSelect={() => setSelectedItem(item)}
+              onSelect={() => {
+                setSelectedItem(item)
+              }}
```

Comment:
What particular rule is asking for this change? It seems like this shouldn't matter